docs/27_map_css_to_mapbox_converter.md

```
@@ -0,0 +1,641 @@
+
+Legends make your maps meaningful 🔍🧭
+
+—
+
+🧑‍💻 Up Next: Want to make those legends interactive — e.g., click to filter?
+
+➡ Continue to [Chapter 26: Interactive Legend Controls](26_interactive_legend_controls.md)
+
+Legendary mapping skills unlocked! 🗺️🧙‍♂️📚
+
+# Chapter 27: Map CSS to MapBox Converter
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)